src/API/WesternStatesWater.WestDaat.Accessors/ApplicationAccessor.cs

```diff
@@ -327,7 +327,15 @@ private async Task<ApplicationStoreResponseBase> UpdateApplicationSubmission(Wat
 
         await db.SaveChangesAsync();
 
-        return new ApplicationStoreResponseBase();
+        var savedNote = await db.WaterConservationApplicationSubmissionNotes
+            .Include(n => n.User).ThenInclude(user => user.UserProfile)
+            .Where(n => n.Id == note.Id)
+            .SingleAsync();
+
+        return new WaterConservationApplicationSubmissionUpdateResponse
+        {
+            Note = savedNote.Map<ApplicationReviewNote>()
+        };
```

Comment:
```suggestion
        var savedNote = await db.WaterConservationApplicationSubmissionNotes
            .Where(n => n.Id == note.Id)
            .ProjectTo<ApplicationReviewNote>(DtoMapper.Configuration)            
            .SingleAsync();

        return new WaterConservationApplicationSubmissionUpdateResponse
        {
            Note = savedNote
        };
```

I would try to use `ProjectTo` here